STMGeom/inc/BackShielding.hh

```diff
@@ -30,7 +30,7 @@ namespace mu2e {
    double   BPHeight()                 const {return _BPHeight;}
    double   Back_dX()                  const {return _Back_dX;}
    double   Back_dY()                  const {return _Back_dY;}
-
+   double   PipeGap()                  const {return _PipeGap;}
```

Comment:
Can we find a more descriptive name for this? I thought this was for the shield pipe (close to the CRV) for a second but this is for the downstream shield house.